airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.tsx

```diff
@@ -85,7 +85,11 @@ const SetDefaultName: React.FC = () => {
 
   useEffect(() => {
     if (selectedService) {
-      setFieldValue("name", selectedService.name);
+      setTimeout(() => {
```

Comment:
:information_source: I'm not entirely sure why this broke, but this seems to be "expected behavior" that this call won't have any effect if it's done while the form is still in it's initial "initialiation" render, in which case it will just take the `initialValues`. I've solved it in this "quick and dirty" way, since I want to open an issue for a larger refactoring of the service form component, that would also solve this issue in a proper way later on.

Reply:
Should the effect clear the timeout when the selected service changes?

Reply:
Done